src/core/Core.js

```diff
@@ -169,6 +169,8 @@ class Uppy {
         data: file.data,
         progress: {
           percentage: 0,
+          bytesUploaded: 0,
+          bytesTotal: file.data.size || 0,
```

Comment:
I think we should add it here: https://github.com/transloadit/uppy/blob/master/src/generic-provider-views/index.js#L123-L149